src/couch_replicator/test/eunit/couch_replicator_scheduler_docs_tests.erl

```diff
@@ -131,17 +131,22 @@ t_scheduler_docs_total_rows({_Ctx, {RepDb, Source, Target}}) ->
     Body = test_util:wait(
         fun() ->
             case req(get, SchedulerDocsUrl) of
-                {200, #{<<"docs">> := [_ | _]} = Decoded} -> Decoded;
-                {_, #{}} -> wait
+                {200, #{<<"docs">> := [_ | _]} = Decoded} ->
+                    Decoded;
+                {_, #{<<"error">> := Error, <<"reason">> := Reason}} ->
+                    ?debugVal(Error, 100),
+                    ?debugVal(binary_to_list(Reason), 100);
```

Comment:
Did you intend to leave this debugging here?

Reply:
Yes.  Sorry, I did not put a comment there about it, but I mentioned it to @nickva so he was aware of the intention.  The reason is that basically I had trouble with finding out the reason for failure.  Initially I thought the timeout should be increased, but, with the help of these extension, it got clear that something else needs to be done.  So I left the debugging calls there to facilitate similar investigations in the future.  Note that this is only visible in case of failures.